README.mdown

```diff
@@ -18,11 +18,13 @@ Add to `external_scripts.json`:
 ```
 
 Set the `WEBHOOK_TOKEN` environment variable to a secret string of your choice
-when running the bot. It will be used in the incoming webhook URL.
+when running the bot. It will be used in the incoming webhook URLs.
+
+### Generic hooks
```

Comment:
The level is off by one here, but I also don't see why Usage is not the second section after Installation anymore. What follows after the installation is how to use any of the supported hooks.